docs/playbook/transforming-data/using-aws-glue/002-deploy-glue-jobs.md

```diff
@@ -16,95 +16,129 @@ If you are still testing or prototyping your script, it is recommended that you
 
 ### 1. Add your script to the Data Platform Project using the GitHub UI
 
-:::note
-If your Glue job was created using the Glue Studio UI,
-you can find the automatically generated script by navigating to the `Script` tab when viewing your job in Glue Studio.
+1. Navigate to the [jobs directory][jobs-directory] in the Data Platform Project and open your department folder.
+1. Click `Add file` and then `Create new file`.
+1. Name your file.
+   This should be the same name you gave your job in Glue Studio, but should be with all **lowercase letters**, **words separated by underscores**, and with the extension **`.py`** at the end.
+   e.g. `address_cleaning_housing_repairs.py`.
+   - _Remember this name as you will need it for the following section._
+1. Open your Glue job in Glue Studio and copy the auto generated **Python** script by navigating to the `Script` tab.
+
+_Note: This is only applicable if your Glue job was created using the Glue Studio UI._
```

Comment:
Maybe just change this to "If you created your job in Glue Studio, ... " instead of having the note underneath